src/oci.c

```diff
@@ -523,6 +526,90 @@ cc_oci_attach (struct cc_oci_config *config,
 }
 
 /*!
+ * Get the home directory for the workload user
+ *
+ * \param config \ref cc_oci_config.
+ *
+ * \return home dir for user on sucess, "/" by default
```

Comment:
This function returns "/" by default. But I think I am going to change this in my next commit.